.github/workflows/run_benchmarks.yml

```diff
@@ -0,0 +1,109 @@
+name: "Launching benchmarks"
+
+on: 
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'     
+        required: true
+        default: 'warning'
+      tags:
+        description: 'Test scenario tags'  
```

Comment:
Change this to:
`on: workflow_dispatch`

Reply:
Done